packages/create-llama/templates/types/streaming/express/src/controllers/chat.controller.ts

```diff
@@ -47,14 +47,29 @@ export const chat = async (req: Request, res: Response) => {
 
     const response = await chatEngine.chat(
       lastMessageContent as MessageContent,
-      messages,
+      messages as ChatMessage[],
       true,
     );
 
     // Transform the response into a readable stream
-    const stream = LlamaIndexStream(response);
+    const { stream, data: streamData } = LlamaIndexStream(response, {
+      parserOptions: {
+        image_url: data?.imageUrl,
+      },
+    });
 
-    streamToResponse(stream, res);
+    // Pipe LlamaIndexStream to response
+    const processedStream = stream.pipeThrough(streamData.stream);
```

Comment:
would it make sense to do this in `LlamaIndexStream` and just return one value there?